tests/unit/test_embedding_models.py

```diff
@@ -409,3 +407,33 @@ def test_do_embedding(self, monkeypatch, tmp_path, n_processes):
 
         args, _ = fake_h5.concatenate.call_args
         assert len(args[2]) == n_processes
+
+
+@pytest.mark.parametrize(
+    "model_name",
+    [
+        "sentence-transformers/multi-qa-MiniLM-L6-cos-v1",
+        "sentence-transformers/multi-qa-mpnet-base-dot-v1",
+    ],
+)
+def test_embedding_size(model_name):
+    model = SentTransformer(model_name)
+    if model_name == "sentence-transformers/multi-qa-mpnet-base-dot-v1":
+        assert model.dim == 768
+    elif model_name == "sentence-transformers/multi-qa-MiniLM-L6-cos-v1":
+        assert model.dim == 384
```

Comment:
It is a detail, feel free not to merge it, but maybe you could do the test this way: 

```suggestion
@pytest.mark.parametrize(
    ("model_name", "expected_dim"),
    [
        ("sentence-transformers/multi-qa-MiniLM-L6-cos-v1", 384),
        ("sentence-transformers/multi-qa-mpnet-base-dot-v1", 768),
    ],
)
def test_embedding_size(model_name, expected_dim):
    model = SentTransformer(model_name)
    assert model.dim == expected_dim
```

In my opinion, it is easier to add new model to test in the future (avoid long `if/elif/else` statement and to repeat the model names at several places)

Reply:
yes, it is better